src/main/java/frc/robot/subsystems/ClimbPivot.java

```diff
@@ -49,58 +56,85 @@ public enum TargetPositions {
   private final double CLIMB_HOLD_CLIMBED = -0.0705;
   private final double CLIMB_IN_SPEED = -0.75;
 
-  // relative to eachother, likely not accurately zero'ed when obtained.x
+  // positions for relation between motor encoder and WCP encoder
+  // relative to eachother, likely not accurately zero'ed when obtained.
   private static final double MIN_ROTOR_POSITION = -50.45;
   private static final double MAX_ROTOR_POSITION = 14.456;
   private static final double MIN_ENCODER_POSITION = 0.611;
   private static final double MAX_ENCODER_POSITION = 0.915;
 
+  // two status variables
   private boolean isClimbOut = false;
   private boolean isStowed = true;
 
+  // setting the starting target position
   private TargetPositions selectedPos = TargetPositions.STOWED;
   private double maxTargetPos = STOWED_MAX_PRESET;
   private double minTargetPos = STOWED_MIN_PRESET;
   private double holdSpeed = CLIMB_HOLD_STOWED;
+
+  // if moveComplete is true it wont move regardless of if its in range. This is to ensure that
+  // when disabled, when re-enabled it doesnt start moving.
   private boolean moveComplete = true;
+
+  // inTolerenace is used to make sure that we're within a ok range between our min goal and max
+  // goal
   private boolean inTolerance = true;
 
+  // the target speed
   private double setSpeed = 0;
 
-  // alerts
+  // alerts for checking if either motor is or isnt connected
   private final Alert NotConnectedErrorOne =
       new Alert("Climb", "Motor 1 not connected", AlertType.kError);
   private final Alert NotConnectedErrorTwo =
       new Alert("Climb", "Motor 2 not connected", AlertType.kError);
   private final Debouncer notConnectedDebouncerOne = new Debouncer(.1, DebounceType.kBoth);
   private final Debouncer notConnectedDebouncerTwo = new Debouncer(.1, DebounceType.kBoth);
 
+  // here we go
   public ClimbPivot() {
     motorLeft = new TalonFX(Hardware.CLIMB_PIVOT_MOTOR_LEFT_ID);
+    // checking to instantiate the 2nd motor if two are installed. If only one is installed, the 2nd
+    // one is set to void.
```

Comment:
```suggestion
    // one is set to null.
```
Strictly speaking, `void` is a type to indicate there is no value while `null` is a value that indicates nothing. They're pretty similar, but one's a type while another is a value.